app/controllers/users_controller.rb

```diff
@@ -0,0 +1,42 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all  
+  end
+  
+  def show
+    user_id = params[:id]
+    @user = User.find_by(id: user_id)
```

Comment:
Finding a logged in user is good functionality to put in a controller filter because it may be used multiple places in the future.